PusherChatkit.xcodeproj/project.pbxproj

```diff
@@ -753,7 +762,10 @@
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
```

Comment:
I think this is already being set in the `Carthage.xcconfig` file on a per-platform basis

Reply:
I've fixed both in f25e46b5a0fdc1076bd8701bf81879e241ba1bea.